### ISSUE TITLE
NVDA should be able to play an earcon or similar when partial formatting is found.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#6537]
Example:
[test question.docx](https://github.com/nvaccess/nvda/files/572868/test.question.docx)
When using control+arrows to find a word, if there's formatting that spans only part of it, do some earcon. after speech refactor, we might be able to do earcons in parallel with the word being spoken depending on how good we can be at getting indexes.

**Comment by jcsteh** [comment]
Failure to report formatting changes within a word is a technical limitation, not a UX decision. For example, if you turn on font attributes and have a word which is half bold and half not in a web browser, you'll notice that NVDA will read the change mid-word. This isn't possible in MS Word because checking the formatting for every single character is far too slow; it's already slow enough checking every word.

**Comment by derekriemer** [comment]
ah got you.
